src/main/scala/chess/Game.scala

```diff
@@ -102,6 +102,10 @@ case class Game(
   // doesnt seem to be used anywhere
   // def moveString = s"$fullTurnCount${player.fold(".", "...")}"
 
+  def fenTurnCount: Int                 = situation.board.variant.fenTurnCount(turnCount)
```

Comment:
This is just the same as `fullTurnCount`. Even in the case of Monster you don't seem to be overriding it to a different value.
I think this can be removed